hudi-utilities/src/main/java/org/apache/hudi/utilities/transform/ChainedTransformer.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.hudi.utilities.transform;
```

Comment:
This class missed the Apache license header. And it's strange why the Travis is green? Does it not check this rule? cc @vinothchandar

Reply:
@yanghua Fixing it.

Reply:
We can add a new section in the[ write documentation page](http://hudi.apache.org/docs/writing_data.html). The branch of documentation is `asf-site`. So it's OK to describe it in another PR.